.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,27 @@
+name: Feature Request
+description: Thanks for taking the time to fill out this feature request!
```

```suggestion
description: Thanks for taking the time to fill out this feature request!
label: [enhancement]
```
`enhancement` is a default label, see https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels

---

Done